target/linux/ar71xx/base-files/etc/board.d/02_network

```diff
@@ -424,7 +424,8 @@ ar71xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "2:lan:2" "3:lan:3" "4:lan:4"
 		;;
-	tl-wr842n-v2)
+	tl-wr842n-v2|\
+	tl-wr942n-v1)
 		ucidef_set_interfaces_lan_wan "eth1.1" "eth0"
 		ucidef_add_switch "switch0" \
 			"0@eth1" "1:lan:4" "2:lan:1" "3:lan:2" "4:lan:3"
```

Comment:
> Switch ports are checked and mapped correctly accordind to port definition writings on backside of the router, LuCI switch page and LEDs.

OK, thanks for confirmation.

> USB LEDs and UART Tx and Rx pins are hardwired on the board and AFAIK can not be used simultaneously. Is this information useful?

You can override what U-Boot configured, later, in kernel (re-assign GPIO14/15 to UART RX/TX signals), this is not a problem at all.

Question here is what is more important for the user: working USB LEDs or serial console and what we should support by default if there is no way to have both things working at the same time. As the USB LEDs are available on front panel I would prefer to support them, as it's done in vendor _not beta version_ firmware.

Are you sure there are no other GPIO signals available on the PCB which we could assign console to? Is some high resolution photo of the PCB of this device available somewhere on the Internet?